msteams-platform/webhooks-and-connectors/how-to/add-incoming-webhook.md

```diff
@@ -90,9 +93,9 @@ To remove an Incoming Webhook from a Teams channel, follow these steps:
 
 ## Code sample
 
-| Sample Name           | Description | C#    | Node.js   |
+| Sample Name           | Description | C#    |  TypeScript |
 |:---------------------|:--------------|:---------|:--------|
-|Incoming Webhook|This sample code demonstrates how to send card using incoming webhook. |[View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/incoming-webhook/csharp)|[View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/incoming-webhook/nodejs) |
+|Incoming Webhook|This sample code demonstrates how to send card using incoming webhook. |[View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/incoming-webhook/csharp)|[View](https://github.com/OfficeDev/TeamsFx-Samples/tree/dev/incoming-webhook-notification) |
```

Comment:
Is "Dev" the right branch?

Reply:
@HuiHuiWu-Microsoft, please suggest the right branch.

Reply:
Different versions of TTK might have different links to download(Currently we're downloading from tags/v1.0.0 which may be changed to tags/v2.0.0 if any breaking change). So I'd suggest to use 'release' branch since it's our stable branch and all the releases are from this branch.